lib/rucio/core/did_meta_plugins/did_column_meta.py

```diff
@@ -267,7 +267,7 @@ def list_dids(self, scope, filters, did_type='collection', ignore_case=False, li
                     yield {
                         'scope': did.scope,
                         'name': did.name,
-                        'did_type': str(did.did_type),
+                        'did_type': str(did.did_type.name),
```

Comment:
`str()` is not needed anymore, `.name` is already a `str`: https://docs.python.org/3/library/enum.html#enum.Enum.name

```suggestion
                        'did_type': did.did_type.name,
```